Bottom: f33f11bb2c7921abd49bf2c745ecd192b7336e31
Top:    7f72d1f68117455c5f4689e11f5566f956a8ef90
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-12-17 21:38:36 +0000

Change scroll-wheel bindings to match more GTK applications










---

diff --git a/src/gpcb-menu.res.in b/src/gpcb-menu.res.in
index f28edff..598324e 100644
--- a/src/gpcb-menu.res.in
+++ b/src/gpcb-menu.res.in
@@ -25,18 +25,18 @@ Mouse =
     up-shift-ctrl = { Mode(Notify) Mode(Restore) Display(ToggleRubberbandMode) }
   }
   Up = {
-    Zoom(0.8)
-    shift = Scroll(up)
-    ctrl = Scroll(left)
+    Scroll(up)
+    shift = Scroll(left)
+    ctrl = Zoom(0.8)
   }
   Down = {
-    Zoom(1.25)
-    shift = Scroll(down)
-    ctrl = Scroll(right)
+    Scroll(down)
+    shift = Scroll(right)
+    ctrl = Zoom(1.25)
   }
 # If you want zoom to center, do this instead.
-  #Up = { {Zoom(0.8) Center()} }
-  #Down = { {Zoom(1.25) Center()} }
+  #Up = { ctrl = {Zoom(0.8) Center()} }
+  #Down = { ctrl = {Zoom(1.25) Center()} }
 
   scroll-left = {
     Scroll(left)
